localstack-core/localstack/services/ec2/patches.py

```diff
@@ -9,11 +11,43 @@
     InvalidSubnetDuplicateCustomIdError,
     InvalidVpcDuplicateCustomIdError,
 )
+from localstack.utils.id_generator import (
+    ExistingIds,
+    ResourceIdentifier,
+    Tags,
+    localstack_id,
+    localstack_id_manager,
+)
 from localstack.utils.patch import patch
 
 LOG = logging.getLogger(__name__)
 
 
+@localstack_id
+def generate_vpc_id(
+    resource_identifier: ResourceIdentifier,
+    existing_ids: ExistingIds = None,
+    tags: Tags = None,
+) -> str:
+    suffix = "".join(random.choice(string.ascii_letters) for _ in range(17))
+    return f"vpc-{suffix}"
```

Comment:
Here we could return `""`.  And maybe add a comment that moto does the generation?

Reply:
Yes I think @cloutierMat merged recently a PR so moto resource can have the custom id generation. I think this utility is for our LocalStack store resources.